packages/path_provider/path_provider/example/integration_test/path_provider_test.dart

```diff
@@ -70,7 +70,7 @@ void main() {
   ];
 
   for (final StorageDirectory? type in _allDirs) {
-    test('getExternalStorageDirectories (type: $type)', () async {
+    testWidgets('getExternalStorageDirectories (type: $type)', () async {
```

Comment:
I just realized that his is missing `(WidgetTestster tester)`. Probably the cause of the test failures

Reply:
🤦🏻